src/retry-helper.ts

```diff
@@ -0,0 +1,58 @@
+import * as core from '@actions/core'
+
+const defaultMaxAttempts = 3
+const defaultMinSeconds = 10
+const defaultMaxSeconds = 20
+
```

Comment:
we had two places with retry logic, and i needed to add a third.

so i went ahead and added a helper class